src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -34,6 +35,11 @@ function renderWithRoute(component: ReactElement) {
   });
 }
 
+jest.mock('../../utils/location', () => ({
+  getLocationOrigin: jest.fn(),
+}));
```

Comment:
I believe you can just do
```
jest.mock('../../utils/location');
```
This automatically exports all exports with a mocked function.